python/neuroglancer/downsample.py

```diff
@@ -34,8 +33,26 @@ def downsample_with_averaging(array, factor):
         counts[indexing_expr] += 1
     return np.cast[array.dtype](temp / counts)
 
+def downsample_segmentation_2D_4x(data):
```

Comment:
The name "max pooling" might be more appropriate, but this algorithm works only works for 2x2 downsampling. There's no way for the user to specify a factor. I might integrate this so that it's used when downsampling segmentations in 2D (e.g. the first two downsamples of zfish) and then uses striding thereafter.